contracts/Vault.sol

```diff
@@ -12,13 +12,20 @@ contract Vault is ERC4626Metadata, Ownable {
 
     // Mapping address => uint256
     mapping(address => uint256) public marketAllowance;
+
+    // Mapping address => uint256
+    // Unix time until receiver can withdraw from vault
+    mapping(address => uint256) public lockedTime;
```

Comment:
should we add a `getLockedTime` function so that users can see when they can withdraw their tokens?